kubeshop/kubtest
# added postman collection e2e test for API server

Test main API server routes 
- create script 
- get created script
- check if script is on scripts list 
- start script execution 
- wait for success execution 
- get executions - check if execution is on list 
- get execution by id 
- get recent executions - check if execution is on list 

## Changes

-

## Fixes

-

## Checklist
